package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "api-sidecar",
+  "version": "1.0.0",
+  "description": "CENNZnet storage index and HTTP service",
+  "main": "src/index.js",
+  "repository": "ssh://git@github.com/cennznet/api-sidecar",
+  "author": "developers@centrality.ai",
+  "license": "Apache",
+  "scripts": {
+    "api": "ts-node src/api/index.ts",
+    "nft:stats": "ts-node src/scanner/scanNFTStats.ts",
+    "nft:finalizationHead": "ts-node src/scanner/trackfinalizeHead.ts"
+  },
+  "dependencies": {
+    "@cennznet/api": "2.1.0",
+    "@tsconfig/node16": "^1.0.2",
+    "axios": "^0.27.2",
+    "bignumber.js": "^9.0.0",
+    "dotenv": "^8.2.0",
+    "fastify": "^3.22.0",
+    "fastify-mongodb": "^4.1.0",
+    "fastify-plugin": "^3.0.0",
+    "moment": "^2.29.1",
+    "mongoose": "^6.0.9",
+    "prettier": "^2.5.1",
+    "ts-node": "^10.7.0",
+    "tslib": "^2.3.1",
+    "tslog": "^3.3.2",
+    "typescript": "^4.6.2",
```

Comment:
nitpik, some of these deps can be moved to `devDependencies`, e.g `typescript`, `prettier`